packages/integration-tests/src/views/storagePage.ts

```diff
@@ -24,4 +24,18 @@ export const storagePage = {
         expect(hostDisk).to.contain('2');
       });
   },
+   getSkipFormattingCheckbox: () => {
```

Comment:
This function should take the Index of the disk to visit

Reply:
Actually, the ID from the UI is not right.
For every host, the IDs are being repeated, so they are not unique. We should include the index of the host as part of the ID.
![disk-checkbox-id](https://user-images.githubusercontent.com/829045/192241649-6b5ada3b-e866-47e0-97d4-1d25782e32a1.png)

Reply:
New PR for solve this problem: https://github.com/openshift-assisted/assisted-ui-lib/pull/1681